packages/compass-e2e-tests/tests/import-export-connections.test.ts

```diff
@@ -119,6 +127,7 @@ describe('Connection Import / Export', function () {
         await runCompassOnce([
           `--import-connections=${file}`,
           ...passphraseArgs,
+          '--trackUsageStatistics',
```

Comment:
The logging test sets `trackUsageStatistics: true`, which I think is what made this pass before, but makes it harder to work on these tests in isolation. Just passing the flag on the command line ensures that telemetry is always enabled.